WorkloadManagementSystem/Client/SandboxStoreClient.py

```diff
@@ -84,17 +87,23 @@ def uploadFilesAsSandbox( self, fileList, sizeLimit = 0, assignTo = {} ):
       if assignTo[ key ] not in self.__validSandboxTypes:
         return S_ERROR( "Invalid sandbox type %s" % assignTo[ key ] )
 
-    if type( fileList ) not in ( types.TupleType, types.ListType ):
-      return S_ERROR( "fileList must be a tuple!" )
+    if not isinstance( fileList, ( list, tuple ) ):
+      return S_ERROR( "fileList must be a list or tuple!" )
 
     for sFile in fileList:
-      if re.search( '^lfn:', sFile ) or re.search( '^LFN:', sFile ):
-        pass
-      else:
-        if os.path.exists( sFile ):
-          files2Upload.append( sFile )
+      if isinstance( sFile, str ):
+        if re.search( '^lfn:', sFile ) or re.search( '^LFN:', sFile ):
```

Comment:
re.search('^lfn:', sFile, flags=re.IGNORECASE)
Note that this will also accepth 'LfN:' ...